x-pack/legacy/plugins/siem/public/components/ml/influencers/get_host_name_from_influencers.ts

```diff
@@ -5,12 +5,18 @@
  */
 
 export const getHostNameFromInfluencers = (
-  influencers: Array<Record<string, string>>
+  influencers: Array<Record<string, string>>,
+  hostName?: string
 ): string | null => {
   const recordFound = influencers.find(influencer => {
     const influencerName = Object.keys(influencer)[0];
```

Comment:
nit - we can just `Object.entries` to get key and value at the same time

Reply:
I created a `getEntries` function which normalizes this data and optionally returns null for their values which makes all of this logic cleaner and within that function I use the `Object.entries`.

Thanks for the advice and keeping me honest to the ways of the JavaScript